sw/device/lib/pinmux.c

```diff
@@ -80,4 +80,14 @@ void pinmux_init(void) {
   reg_offset = kTopEarlgreyPinmuxMioOutIoc9 << 2;
   mask = PINMUX_MIO_OUTSEL_0_OUT_0_MASK;
   mmio_region_write32(reg32, reg_offset, reg_value & mask);
+
+  // Configure USBDEV SENSE outputs to be high-Z (IOR0, IOR1)
+  reg32 =
+      mmio_region_from_addr(PINMUX0_BASE_ADDR + PINMUX_MIO_OUTSEL_0_REG_OFFSET);
+  reg_value = 2;  // high-Z value
```

Comment:
We should probably add these constants to a header file. Would you mind opening an issue so we can track it? No need to block this PR but if you could add `// TODO(lowRISC/opentitan#XXXXX): replace with named constant` then it will be easy to grep for when the issue is fixed.

Reply:
No problem. Done.